reframe/frontend/executors/policies.py

```diff
@@ -27,6 +22,7 @@ def run_check(self, check, partition, environ):
         )
         task = RegressionTask(check)
         self._tasks.append(task)
+        self.stats.add_task(task)
```

Comment:
This should go in the base class.

Reply:
I am not sure if I remember right, but didn't you say that the policies are not necessarily required to use tasks?